ci/autoscaler/pipeline.yml

```diff
@@ -107,7 +107,7 @@ resources:
   icon: github
   source:
     uri: git@github.com:cloudfoundry/app-autoscaler-release
-    private_key: ((autoscaler_git_key))
+    private_key: ((autoscaler-ci-bot-authentication-key-private))
```

Comment:
Using the authentication ssh-key from `autoscaler-ci-bot` improves the concourse logs in a way that they don't confuse the reader with showing stuff like `kevin.cross@sap.com`. For example cloning the `app-autoscaler-release` via Concourse resource looks now like this

```bash
selected worker: concourse-worker-89676c5bd-k6gz7
Identity added: /tmp/git-resource-private-key (ApplicationAutoscaler@sap.com)
HEAD is now at 42fdb99a sign commits during release creation
```

Reply:
Nice!